products/protractor.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+changelogTemplate: https://github.com/angular/protractor/blob/master/CHANGELOG.md
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
```

Comment:
Please add the corresponding blank line following End Of Life [Front Matter YAML](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#front-matter-yaml)

Reply:
I did add a blank line after the `eolColumn` ..is there anything I missed?

Reply:
done